UN/.htaccess

```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://www.un.org/ [R=301,L]
+RewriteRule ^FAO/?$ https://www.fao.org [R=301,L]
```

Comment:
You can update this one.  Either just commit a correction, or add a commit and squash the two into one commit and force push the result.